src/components/browse/BrowseCard.tsx

```diff
@@ -165,8 +167,29 @@ export default function BrowseCard(props: BrowseCardProps) {
    * Placeholders until we have the actual data
    */
   const pricePerShare = 729.48;
-  const aprFee = '10%';
-  const totalValueLocked = '$379M';
+
+  const [poolStats, setPoolStats] = useState<PoolStats>();
+
+  useEffect(() => {
+    let mounted = true;
+    const fetchPoolStats = async () => {
+      const response = await axios.get(
+        `http://34.94.221.78:3000/pool_stats/${blendPoolMarkers.poolAddress}/1`
```

Comment:
Let's move the API's base url to a constant so it's easy to change

Reply:
Of course!

Reply:
@adamegyed told me there's a way to put this into an env var on Netlify, so let's actually do that instead. Adam can you provide some guidance here?